### PR TITLE
Update "View own rates" cuke for ng work packages impl.

### PR DESCRIPTION
**Pull request by myabc**
https://www.openproject.org/work_packages/5225

**PLEASE MERGE IN CONJUNCTION WITH opf/openproject#1237**

NOTE: ng L10n implementation currently incomplete (thus EUR symbol
is displayed before rate value).

### CODE REVIEW

File: features/view_own_rates.feature [hunk 1]
```diff
@@ -68,16 +68,9 @@ Feature: Permission View Own hourly and cost rates
...
-      | Labor costs    |
-      | Unit costs     |
-    And I follow "Apply"
-    Then I should see "24.00 EUR"
-    And I should see "10.00 EUR"
-    And I should see "14.00 EUR"
-    And I should not see "33.00 EUR" # labour costs only of Manager
-    And I should not see "35.00 EUR" # material costs only of Manager
-    And I should not see "43.00 EUR" # labour costs of me and Manager
-    And I should not see "49.00 EUR" # material costs of me and Manager
```
Comment (ulferts) [inline features/view_own_rates.feature:82]: Why did you remove those "I should not see" lines?

Comment (myabc) [inline features/view_own_rates.feature:82]: @ulferts sorry, that's a mistake. I'll revise the PR.
